### PR TITLE
Cypress Dev Environment

### PR DESCRIPTION
**Pull request by ksmontville**
This PR sets up the infrastructure for executing Cypress tests on development Firebase projects.

### CODE REVIEW

**Review by Emily-ejag — CHANGES_REQUESTED** [review]
LGTM I think wit will safe soo much time, can you solve the conflicts?

**Review by Emily-ejag — APPROVED** [review]

File: .github/workflows/deploy-firebase-test-cypress.yml [hunk 1]
```diff
@@ -33,18 +31,28 @@ jobs:
...
     timeout-minutes: 90
     strategy:
       fail-fast: false
       matrix:
         browser: [chromium, edge]
-        containers: [1, 2, 3]
+        containers: [1, 2, 3, 4]
     env:
-      CYPRESS_BASE_URL: ${{ needs.build-and-preview.outputs.deployUrl }}
+      #      CYPRESS_BASE_URL: ${{ needs.build-and-preview.outputs.deployUrl }}
```
Comment (Emily-ejag) [inline .github/workflows/deploy-firebase-test-cypress.yml:55]: Is it because of commenting on this that it is not deploying?

**Review by Emily-ejag — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-firebase-test-cypress.yml (hunk 1 above)
Comment (ksmontville) [reply .github/workflows/deploy-firebase-test-cypress.yml]: Yes I don't expect the tests to actually deploy and run yet; still working on the GitHub action to interact with the dev server.

File: src/pages/ProgressReport.vue [hunk 2]
```diff
@@ -13,7 +13,7 @@
             <div>
               <div class="uppercase font-light text-gray-500 text-md">Administration</div>
               <div class="administration-name uppercase">
-                {{ getTitle(administrationInfo, isSuperAdmin) }}
+                {{ publicName }}
```
Comment (richford) [inline src/pages/ProgressReport.vue:16]:
```suggestion
                {{ displayName }}
```

File: src/pages/ProgressReport.vue [hunk 3]
```diff
@@ -145,6 +145,13 @@ const reportViews = [
   { name: 'Score Report', constant: false },
 ];
 
+const publicName = computed(() => {
```
Comment (richford) [inline src/pages/ProgressReport.vue:148]:
```suggestion
const displayName = computed(() => {
```

File: src/pages/ScoreReport.vue [hunk 4]
```diff
@@ -19,7 +19,7 @@
                 <div>
                   <div class="uppercase font-light text-gray-500 text-sm">Administration</div>
                   <div class="administration-name mb-4">
-                    {{ _toUpper(getTitle(administrationInfo, isSuperAdmin)) }}
+                    {{ _toUpper(publicName) }}
```
Comment (richford) [inline src/pages/ScoreReport.vue:22]:
```suggestion
                    {{ _toUpper(displayName) }}
```

File: src/pages/ScoreReport.vue [hunk 5]
```diff
@@ -355,6 +355,13 @@ const props = defineProps({
 
 const initialized = ref(false);
 
+const publicName = computed(() => {
```
Comment (richford) [inline src/pages/ScoreReport.vue:358]:
```suggestion
const displayName = computed(() => {
```

**Review by richford — CHANGES_REQUESTED** [review]

**Review by lucasxsong — APPROVED** [review]

**Pull request merged**